src/web/mjs/engine/Storage.mjs

```diff
@@ -720,6 +720,14 @@ export default class Storage {
         } );
     }
 
+    /**
+     * Helper function to generate the path where the bookmarks and markers are stored.
+     */
+    _bookmarkOutputPath( ) {
```

Comment:
May make this a `getter`, so it can be used like a member and not like a function